build.gradle

```diff
@@ -126,10 +126,15 @@ ext.publishingRepository = version.toString().endsWith("-SNAPSHOT") ? snapshotRe
 
 configurations {
     mps
+    jackson
 }
 
 dependencies {
     mps "com.jetbrains:mps:$mpsVersion"
+    jackson 'com.fasterxml.jackson.core:jackson-core:2.12.3'
```

Comment:
This needs to be `2.11.*` for MPS 2020.3 since that's the version used by MPS internally to avoid strange errors at runtime.

Reply:
Then we should just use the jars in MPS and add a solution containing the stubs for it instead of downloading them again. @dbinkele On the "Java" tab remove the jars from the "Libraries" and on the "Common" tab add the jackson jars in the "/lib" folder of MPS. MPS will use relative paths ("$MPS_HOME$/lib/...") if you do that.

Then the MPS-extensions project is the right place for it.

Reply:
OK ... understood. But I had such a nice download solution. It seem the Apache stuff are checked in jars. Then it would make sense to download those...?!

Reply:
Yes

Reply:
@slisson Please give me a hint how to add those jars in $MPSHOME/lib in this dialog 
![image](https://user-images.githubusercontent.com/50654800/116681205-f3854a80-a9ac-11eb-94ac-ec738b06dba2.png)

Reply:
@coolya Some of the Apache libs checked into the repo can be replaced by later versions which are found in the $MPS_HOME/lib folder! No major version changes only minor. Makes probably sense to use those instead of the older ones!?

Reply:
You have to add a different model root first.

Reply:
Pointing to which folder?

Reply:
Macros are applied automatically. Just choose the lib folder and MPS will write $MPS_HOME$/lib to the file.

Reply:
@slisson Please have a look once more...

Reply:
The download of the jackson jars is not required anymore. Revert all changes to the build.gradle file.